service/src/main/kotlin/fi/espoo/evaka/vasu/VasuQueries.kt

```diff
@@ -61,3 +61,25 @@ fun Database.Transaction.updateVasuDocument(id: UUID, content: VasuContent) {
         .bind("content", content)
         .updateExactlyOne()
 }
+
+fun Database.Read.getVasuDocumentSummaries(childId: UUID): List<VasuDocumentSummary> {
+    // language=sql
+    val sql = """
+        SELECT 
+            vd.id,
+            vt.name AS name,
+            vc.updated AS modified_at,
```

Comment:
at some point we need to add a new db column for this since `updated` is automatically changed e.g. by migrations

Reply:
Added a todo to board